extras/rya.export/export.api/src/main/xsd/MergeConfiguration.xsd

```diff
@@ -66,15 +72,23 @@ under the License.
       </xs:restriction>
     </xs:simpleType>
 
+    <xs:simpleType name="CopyType">
+      <xs:restriction base="xs:string">
+        <xs:enumeration value="FileExport"/>
+        <xs:enumeration value="FileImport"/>
+        <xs:enumeration value="ConnectedDatastores"/>
+      </xs:restriction>
+    </xs:simpleType>
+
     <xs:simpleType name="Port">
       <xs:annotation>
         <xs:documentation>
-          Port number in the range [1, 65536].
+          Port number in the range [0, 65535].
```

Comment:
don't use port 0.  if you provide 0 to java it generates a random system port number and then we can't really connect back into it